test/test-ipc-send-recv.c

```diff
@@ -100,12 +123,22 @@ static int run_test(void) {
 
   spawn_helper(&ctx.channel, &process, "ipc_send_recv_helper");
 
+  // Perform two writes to the same pipe to make sure that on Windows we are
+  // not running into issue 505:
+  //   https://github.com/libuv/libuv/issues/505
```

Comment:
If there is a better way to test this, let me know.  @mad-rain's code was basically creating two write requests for the same pipe and the second one would fail at `src/win/pipe.c#L1637`.  I just took his reproduction recipe, since it was almost identical to `ipc_send_recv_tcp` and refactored it to do two writes instead of one.